docker-compose.yml

```diff
@@ -88,7 +88,7 @@ services:
       PRIMARY_HOSTNAME: ${HOST_NAME?missing main hostname (e.g., localhost for development)}
       INTERNAL_HOSTNAMES: "web"
       DEBUG: ${DEBUG:-0}
-      SOURCE_REPO_URL: ${SOURCE_REPO_URL:?missing source repository URL}
+      SOURCE_REPO_URL: ${SOURCE_REPO_URL}
```

Comment:
It is better to have a safeguard because `SOURCE_REPO_URL` is required for the proper functionality of the services.